src/topology/perfect.lean

```diff
@@ -211,3 +218,342 @@ begin
 end
 
 end kernel
+end basic
+
+section scheme
+
+/-- A `β`-scheme on `α` is a function from `list β` to `set α`.
+We typically think of this as a "tree" of subsets of `α`, and use the appropriate terminology
+(branch, children, etc.).
+The usefulness of such a scheme is that a map `(ℕ → β) → α` can often be thought of as
+a sort of "limiting object" of a `β`-scheme on `α`. -/
+def set.scheme (β α : Type*) := list β → set α
```

Comment:
```suggestion
@[derive inhabited]
def set.scheme (β α : Type*) := list β → set α
```
then you don't need the instance below

Reply:
Not needed since `set.scheme` is now not defined.